src/main/java/se/sundsvall/billingdatacollector/integration/opene/kundfakturaformular/mapper/ExternalMapper.java

```diff
@@ -0,0 +1,166 @@
+package se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.mapper;
+
+import static java.util.Optional.ofNullable;
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import static se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.mapper.KundfakturaformularMapper.APPROVED_BY;
+import static se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.mapper.KundfakturaformularMapper.CATEGORY;
+import static se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.mapper.KundfakturaformularMapper.INVOICE_DESCRIPTION;
+import static se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.mapper.KundfakturaformularMapper.MAX_DESCRIPTION_LENGTH;
+import static se.sundsvall.billingdatacollector.integration.opene.util.XPathUtil.extractValue;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.model.ExternFaktura;
+import se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.model.OpeneCollections;
+import se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.model.external.AktivitetskontoExtern;
+import se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.model.external.AnsvarExtern;
+import se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.model.external.BerakningExtern;
+import se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.model.external.MomssatsExtern;
+import se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.model.external.ObjektkontoExtern;
+import se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.model.external.UnderkontoExtern;
+import se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.model.external.VerksamhetExtern;
+import se.sundsvall.billingdatacollector.model.BillingRecordWrapper;
+
+import generated.se.sundsvall.billingpreprocessor.AccountInformation;
+import generated.se.sundsvall.billingpreprocessor.AddressDetails;
+import generated.se.sundsvall.billingpreprocessor.BillingRecord;
+import generated.se.sundsvall.billingpreprocessor.Invoice;
+import generated.se.sundsvall.billingpreprocessor.InvoiceRow;
+import generated.se.sundsvall.billingpreprocessor.Recipient;
+import generated.se.sundsvall.billingpreprocessor.Status;
+import generated.se.sundsvall.billingpreprocessor.Type;
+
+@Component
```

Comment:
If MapperHelper doesn't have to be a @Component, then this class doesn't need to be either...